tests/src/core/geometry/CMakeLists.txt

```diff
@@ -38,5 +35,9 @@ set(TESTS
   )
 
 foreach(TESTSRC ${TESTS})
-  add_qgis_test(${TESTSRC} MODULE core LINKEDLIBRARIES qgis_core)
+  add_qgis_test(${TESTSRC} MODULE core_geometry LINKEDLIBRARIES qgis_core)
```

Comment:
I'm not a big fan of this change. @3nids what do you think?

Reply:
@nyalldawson This is "required" to have a simple regex for geometry classes for coverage build.
Since I have splitted QgsGeometryTest, one could only test dedicated classes test_core_linestring, test_core_point, etc. so with this change one can run "sub-tests" with core_geometry* regex.

Reply:
isn't this backwards though?

Reply:
@nyalldawson Sorry, but I don't understand, backwards of what ?

Reply:
aren't you tagging EVERY test except the geometry ones as "core_geometry"?

Reply:
The command `ctest -R core_geometry` executes 22 tests including 21 in the folder `core/geometry` and 1 for `test_core_geometryimport`

Reply:
Ah, sorry, should have looked closer at the wider context here!